cmd/simulacra/README.md

```diff
@@ -22,7 +22,54 @@ Simulacra can be configured through command line for quick operations, or a conf
 
 This command invokes Simulacra to spin up a VM, install the Ops Agent on it and then install the required third party applications. Based on the Ops Agent config in the example above, Simulacra will install Redis on the created VM. 
 
-## Command Line Arguments
+**Note:** We need to pass in --tags=integration_test every time we run Simulacra. This is because Simulacra uses the Ops Agent Integration Test library internally and this library requires this flag. 
+
+## Prerequisites
+
+Before we can run Simulacra, we need to perform these following steps:
+
+1. Set up gcloud authentication.
+```
+> gcloud auth login
+
+```
+2. Run gcloud application-default login so that Simulacra is able to access Google APIs as you.
+```
+> gcloud auth application-default login
+
+```
+3. Set the 'ZONES' environment variable. This is a requirement due to the integration testing library that we use interally. The value can be any **valid** GCP zone. This value does not impact the zone where your VM will be created. 
+```
+> export ZONES=us-central1-b
+```
+
+4. Create a [cloud storage bucket](https://cloud.google.com/storage/docs/buckets) on GCP, if you don't already have one. You can use Pantheon to do this. 
```

Comment:
Does this have to be in the same project we're creating a VM in? And lets document here what its used for. IIRC, this isn't necessary if w're not providing a custom Ops Agent config right?